### PR TITLE
saksham: Remove invalid gitter link

### PR DESCRIPTION
**Pull request by sils**

### CODE REVIEW

File: _reports/saksham.md [hunk 1]
```diff
@@ -31,10 +31,7 @@ social:
       - link: https://github.com/anctartica
   - GitLab:
       - username: anctartica
-      - link: https://gitlab.com/anctartica
-  - Gitter:
-      - username: anctartica
-      - link: https://gitter.im/anctartica
+      - link: https://gitlab.com/anctartic
```
Comment (saksham189) [inline _reports/saksham.md]: `a` at the end

**Comment by jayvdb** [comment]
@saksham189 why not rename your gitlab account too?

Then we can change them all to be consistent

**Comment by saksham189** [comment]
yes. renamed that too.

File: _reports/saksham.md [hunk 2]
```diff
@@ -32,9 +32,6 @@ social:
   - GitLab:
       - username: anctartica
       - link: https://gitlab.com/anctartica
-  - Gitter:
-      - username: anctartica
-      - link: https://gitter.im/anctartica
```
Comment (saksham189) [inline _reports/saksham.md]: i have renamed all my accounts from anctartica to saksham189 so, we need only to update all the links rather than removing.

**Comment by utkarsh2102** [comment]
Hey,
This needs to be fixed at the earliest.
@sils, could you please update the PR?

**Comment by jayvdb** [comment]
ack fa09436

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**